x-pack/plugins/enterprise_search/server/routes/workplace_search/role_mappings.ts

```diff
@@ -17,6 +17,21 @@ const roleMappingBaseSchema = {
   authProvider: schema.arrayOf(schema.string()),
 };
 
+export function registerOrgEnableRoleMappingsRoute({
+  router,
+  enterpriseSearchRequestHandler,
+}: RouteDependencies) {
+  router.post(
+    {
+      path: '/api/workplace_search/org/role_mappings/enable_role_based_access',
+      validate: false,
+    },
+    enterpriseSearchRequestHandler.createRequest({
+      path: '/ws/org/role_mappings/enable_role_based_access',
```

Comment:
[Not a change request]

I'm a little confused as to why this API endpoint has to be namespaced when clicking "enable RBAC" does so for both AS & WS. Nevertheless, this works great!

Reply:
I had the same point but was told it was going to be kept separate 🤷